py3status/modules/kdeconnector.py

```diff
@@ -119,6 +119,17 @@ def _get_device_id(self, bus):
 
         return None
 
+    def _get_isTrusted(self):
+        try:
+            # New method which replaced 'isPaired' in version 1.0
+            return self._dev.isTrusted
+        except Exception:
```

Comment:
This should probably be `AttributeError` rather than `Exception` just to be more specific.